resources/crds/hammerdb_crds/mssql/ripsaw_v1alpha1_hammerdb_mssql_vm.yaml

```diff
@@ -90,10 +90,16 @@ spec:
         extra_options:
           - none
           #- hostpassthrough
-        # PVC
-        #storageclass: ocs-storagecluster-ceph-rbd
+        ## OCS PVC
+        pvc: false # enable for OCS PVC
+        pvc_storageclass: ocs-storagecluster-ceph-rbd
         # Can be one of ReadWriteOnce,ReadOnlyMany,ReadWriteMany Default: ReadWriteOnce
-        #pvcaccessmode: ReadWriteMany
+        pvc_pvcaccessmode: ReadWriteMany
         # Can be one of Filesystem,Block Default: Filesystem
-        #pvcvolumemode: Block
-        #storagesize: 5Gi
+        pvc_pvcvolumemode: Block
+        pvc_storagesize: 10Gi
+        ## HostPath - Configuring SELinux on cluster workers
+        hostpath: false # enable for hostpath
+        hostpath_path: /var/tmp/disk.img
+        hostpath_storagesize: 10Gi
```

Comment:
Hey @ebattat  I'm still missing some documentation for all these fields

Reply:
Done https://github.com/cloud-bulldozer/benchmark-operator/pull/596/commits/94e03e13df29554a918f2ca50172400a23cd032c